doc/source/admin/reports_options.rst

```diff
@@ -149,7 +149,7 @@
 
 :Description:
     Mail
-:Default: your_bugs@bx.psu.edu
+:Default: ``your_bugs@bx.psu.edu``
```

Comment:
This isn't right - but it isn't a problem with this PR at all. I fell into a rabbit hole trying to track things down - Galaxy just isn't consistent about this so I created https://github.com/galaxyproject/galaxy/issues/5751.